content/authors/admin/_index.md

```diff
@@ -31,44 +31,4 @@ social:
 user_groups: [Root]
 
 ---
-2i2c is a non-profit organization dedicated to supporting tools and workflows
```

Comment:
Note - this was cruft from an old version of the site that wasn't being displayed, so I'm just removing it to avoid confusion